lib/blueprinter/view.rb

```diff
@@ -38,6 +43,10 @@ def exclude_fields(field_names)
       end
     end
 
+    def *(custom_transformer)
+      transformers << custom_transformer
+    end
```

Comment:
I've never seen `*` used for anything but multiplication. Is this common style?

Otherwise, can we come up with a method name that makes clear that it is appending transformer here?
I'm worried we will run out of symbols to define new methods 🤣

Reply:
I am usually not into operator overloading, but then I was inspired by the method written for adding fields into the view
def <<(field)
      fields[field.name] = field
end
I was under the impression that its a style blueprinter follows for adding setters. I agree to all the reviewers @philipqnguyen  and @mcclayton  when you feel that we can run out of operators to overload. For now I will change the method name to transform :)

Reply:
can you rename to `add_transformer`?  Otherwise it seems like `view.transform` is already invoking the transformer, which is not entirely accurate since it's just storing it to be called later.